config.js

```diff
@@ -8,12 +8,14 @@ exports.config = {
      * 
      * Default: 10 Seconds.
     */
-    "change_interval" : 10,
+    "change_interval" : 1,
```

Comment:
Shouldn't this be default to 10 seconds?

Reply:
thats my fault 🤦‍♂